ga/README.en.md

```diff
@@ -0,0 +1,46 @@
+# Parameter search by Genetic Algorithm
+
+[日本語](README.md) | English
+
+## Overview
+
+Contains a set of source code to perform parameter search of the urn-model using the Genetic Algorithm (GA).
+
+## What is actually done
+
+### Flow
+
+Parameter search by genetic algorithm is carried out in the following steps:
+
+1. randomly generate a population with a vector representing the parameters: `(rho, nu, friendship, recentness)` for each individual. 2.
+2. run the pot model based on the parameters, calculate the goodness of fit from the results and record this
```

Comment:
pot should say urn

Reply:
I think

Reply:
resolved https://github.com/tsukuba-websci/MPS-TOM-Urnmodel/pull/26/commits/7cd7f15961ab9c51c16eb4ad06c610bbd7b54b53